pkg/controller/map/storage/handler/ocp/handler.go

```diff
@@ -68,6 +69,7 @@ func (r *Handler) changed(storageClass *ocp.StorageClass) {
 	err := r.List(context.TODO(), &list)
 	if err != nil {
 		err = liberr.Wrap(err)
+		log.Error(err, "failed to list StorageMap CRs")
```

Comment:
does the linter look for logging along with errors?
@liranr23 ^ related to what we discussed on the PR that fixed the calls to liberr.Wrap

Reply:
I see, makes sense